tests/scripts/kubeadm-install.sh

```diff
@@ -5,6 +5,18 @@ KUBE_VERSION=${1:-"v1.7.5"}
 null_str=
 KUBE_INSTALL_VERSION="${KUBE_VERSION/v/$null_str}"-00
 
+ # init flexvolume
+if [[ $KUBE_VERSION == v1.5* ]] || [[ $KUBE_VERSION == v1.6* ]] || [[ $KUBE_VERSION == v1.7* ]] ;
```

Comment:
What about upcoming K8s versions? Does this need to be modified per K8s version?

Reply:
For 1.8.x this is not needed as the flexvolume is automatic discovered. https://github.com/kubernetes/community/pull/833.